{
    "body": "@adamjstewart @darmac \r\nhttps://github.com/spack/spack/pull/21402\r\nAn error occurred with the changed checksum value on this PR. So I returned the checksum value. I have confirmed that I can get the package successfully in all versions.\r\nPlease let me know if you have any other intentions to modify the PR.",
    "user": "t-karatsu",
    "url": "https://api.github.com/repos/spack/spack/issues/21829",
    "updated_at": "2021-02-21 12:16:35",
    "created_at": "2021-02-21 10:48:20",
    "closed_at": "2021-02-21 11:46:56",
    "state": "closed",
    "title": "py-sip: Fixed checksum error.",
    "number": 21829,
    "milestone": null,
    "labels": [],
    "id": 812825656,
    "html_url": "https://github.com/spack/spack/pull/21829",
    "assignees": [],
    "comments": 1
}